tests/test_query_engine.py

```diff
@@ -31,7 +31,11 @@ def test_run_generated_sql_get_single_column_default_population(
     database, setup_test_database
 ):
     input_data = [
-        *patient(1, ctv3_event("Code1")),
+        *patient(
+            1,
+            RegistrationHistory(StartDate="2000-01-01", EndDate="2999-12-31"),
```

Comment:
Why add this?

Reply:
Just for coverage; the mock backend schema's `patient` function adds a default RegistrationHistory if one isn't included in the entities, but no test did that.  I left it in and added a test that used it in case it ewas useful later, but I guess it makes more sense to remove the `if` from the `patient` function instead